crates/red_knot_python_semantic/src/types.rs

```diff
@@ -3229,3 +3230,155 @@ mod tests {
         Ok(())
     }
 }
+
+#[cfg(test)]
+mod property_tests {
+    use super::tests::{setup_db, Ty};
+    use quickcheck::{Arbitrary, Gen};
+    use quickcheck_macros::quickcheck;
+
+    impl Arbitrary for Ty {
+        fn arbitrary(g: &mut Gen) -> Ty {
+            match u32::arbitrary(g) % 33 {
```

Comment:
There must be a better way to do this. If only we had the author of this library on our team...